src/logic/calculate.test.js

```diff
@@ -0,0 +1,19 @@
+import calculate from './calculate';
+
+describe('Calculator', () => {
+  it('should clear the calculator data when AC button is clicked', () => {
+    const obj = {
+      total: '100',
+      next: '50',
+      operation: '+',
+    };
```

Comment:
You have done great at writing the tests for the operate.js and calculate.js modules. Kindly ensure that all functionalities & scenarios of the calculate.js modules are covered.